### ISSUE TITLE
[rust/en] Correction about slices

### ISSUE THREAD

**Issue by carlomilanesi** [adambard/learnxinyminutes-docs#2268]
Rust slices are intervals, not simple positions.
 Therefore, the following sentence about slices:

```
// This is basically an immutable pointer to a string – it doesn’t
// actually contain the contents of a string, just a pointer to
// something that does (in this case, `s`)

```

should become:

```
// This is basically an immutable pair of pointers to a string – it doesn’t
// actually contain the contents of a string, just a pointer to
// the begin and a pointer to the end of a string buffer,
// statically allocated or contained in another object (in this case, `s`)

```

**Comment by vendethiel** [comment]
Thanks for the clarification. Would you mind sending a pull request to fix it? :)

**Comment by divayprakash** [comment]
Fixed in 08c653a under #2269.
